app/src/main/java/com/dessalines/thumbkey/keyboards/CommonKeys.kt

```diff
@@ -329,6 +330,7 @@ val RETURN_KEY_ITEM =
                 color = ColorVariant.SECONDARY,
             ),
         backgroundColor = ColorVariant.SURFACE_VARIANT,
+        longPress = KeyAction.CommitText("\n"),
```

Comment:
These are hidden actions, so they need to be documented in some way. Add them both to the guide in the README.